File: Helpers/src/main/java/com/dg/helpers/CollectionHelper.java
package com.dg.helpers;

import java.util.Collection;

/**
 * Created by Daniel Cohen Gindi (danielgindi@gmail.com)
 */
public class CollectionHelper
{
    public static boolean isEmpty(Collection array)
    {
        return array == null || array.isEmpty();
    }

    public static boolean isEmpty(Object[] array)
    {
        return array == null || array.length == 0;
    }
}

File: Helpers/src/main/java/com/dg/helpers/CollectionHelper.kt
package com.dg.helpers

@Suppress("unused")
object CollectionHelper
{
    fun isEmpty(array: Collection<*>?): Boolean
    {
        return array == null || array.isEmpty()
    }

    fun isEmpty(array: Array<Any>?): Boolean
    {
        return array == null || array.isEmpty()
    }
}
